storagemarket/provider_test.go

```diff
@@ -1103,6 +1103,61 @@ func TestDealVerification(t *testing.T) {
 	}
 }
 
+func TestIPNIAnnounce(t *testing.T) {
```

Comment:
The "announce" and "don't announce" subtests have almost identical code, so I'd suggest that we refactor them so that each subtest calls the same method with a boolean parameter indicating whether to announce or not. Take a look at TestSimpleDealHappy for an example

Reply:
Resolved in [be5958e](https://github.com/filecoin-project/boost/pull/1051/commits/be5958e0b657e897f70c73589710d16557cb71ef)